main/src/main/scala/sbt/internal/SettingsGraph.scala

```diff
@@ -14,38 +14,26 @@ import sbt._
 import sbt.internal.io.Source
 import sbt.internal.util.AttributeMap
 import sbt.internal.util.complete.Parser
-import sbt.io.Glob
+import sbt.io.syntax._
+import sbt.nio.FileStamper
+import sbt.nio.Keys._
+import sbt.nio.file.Glob
 
 import scala.annotation.tailrec
 
-object TransitiveGlobs {
-  val transitiveTriggers = Def.taskKey[Seq[Glob]]("The transitive triggers for a key")
-  val transitiveInputs = Def.taskKey[Seq[Glob]]("The transitive inputs for a key")
-  val transitiveGlobs =
-    Def.taskKey[(Seq[Glob], Seq[Glob])]("The transitive inputs and triggers for a key")
-}
-private[sbt] object InputGraph {
-  @deprecated("Source is also deprecated.", "1.3.0")
+private[sbt] object SettingsGraph {
```

Comment:
There already is a `SettingGraph`, so this is a little confusing...

Reply:
I can’t argue with that. I am actually hoping/planning to get rid of this anyway. I understand the problem better than when I first introduced this.